src/main/java/com/commercetools/sync/types/utils/TypeUpdateActionUtils.java

```diff
@@ -0,0 +1,93 @@
+package com.commercetools.sync.types.utils;
+
+
+import com.commercetools.sync.commons.exceptions.BuildUpdateActionException;
+import com.commercetools.sync.types.TypeSyncOptions;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.types.Type;
+import io.sphere.sdk.types.TypeDraft;
+import io.sphere.sdk.types.commands.updateactions.ChangeName;
+import io.sphere.sdk.types.commands.updateactions.SetDescription;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Optional;
+
+import static com.commercetools.sync.commons.utils.CommonTypeUpdateActionUtils.buildUpdateAction;
+import static com.commercetools.sync.types.utils.TypeUpdateFieldDefinitionActionUtils.buildFieldDefinitionsUpdateActions;
+import static java.lang.String.format;
+import static java.util.Collections.emptyList;
+
+public final class TypeUpdateActionUtils {
+
+    /**
+     * Compares the {@code name} values of a {@link Type} and a {@link TypeDraft}
+     * and returns an {@link Optional} of update action, which would contain the {@code "changeName"}
+     * {@link UpdateAction} if values are different.
+     *
+     * @param oldType the type that should be updated.
+     * @param newType the type draft which contains the new name.
+     * @return optional containing update action or empty optional if names are identical.
+     */
+    @Nonnull
+    public static Optional<UpdateAction<Type>> buildChangeNameAction(
+        @Nonnull final Type oldType,
+        @Nonnull final TypeDraft newType) {
+
+        return buildUpdateAction(oldType.getName(), newType.getName(),
+            () -> ChangeName.of(newType.getName()));
+    }
+
+
+    /**
+     * Compares the {@link LocalizedString} descriptions of a {@link Type} and a {@link TypeDraft} and
```

Comment:
The javadoc here doesn't seem to be written with a consistent style to that of the `buildChangeNameAction` method. Here you say: "Compares the {@link LocalizedString} descriptions" but in the name method, you didn't mention it in the same way, eventhough name is also a localized string. I was a bit confused to the point that I though that name is not a localized string. Please write them in a consistent way :)

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/304/commits/9801dbd87fa39ffda80ead4f33a0fac5f7103070#diff-bf9ae45f65a3d1650823f7c0e7669629R24